Tests/Rules/UseCorrectCasing.tests.ps1

```diff
@@ -38,4 +38,11 @@ Describe "UseCorrectCasing" {
         }
         Invoke-Formatter 'invoke-dummyFunction' | Should -Be 'Invoke-DummyFunction'
     }
+
+    It "preserves script paths" -Skip:($IsLinux -or $IsMacOS) {
+        $uncPath = [System.IO.Path]::Combine("\\$(HOSTNAME.EXE)\C$\", $TestDrive, "$([guid]::NewGuid()).ps1")
```

Comment:
It doesn't seem like this is a problem for Windows only, we should probably run this on Linux/Mac too:
```powershell
if ( $IsWindows ) {
$exePath = [System.IO.Path]::Combine("\\$(HOSTNAME.EXE)\C$\", $TestDrive, "$([guid]::NewGuid()).ps1")
}
else {
$exePath = "/bin/ls"
}
$scriptDefinition = ". $exePath"
```

Reply:
I added a general test for paths that runs on all platforms and kept my UNC path test as a special test. I don't understand how `/bin/ls` would be useful, this issue about UNC script paths that get interpreted by `Get-Command` as type `ExternalScript`